model_analyzer/reports/report_manager.py

```diff
@@ -917,8 +917,13 @@ def _build_detailed_table(self, model_config_name):
             reverse=True)
         cpu_only = model_config.cpu_only()
 
-        first_column_header = 'Request Concurrency' if self._mode == 'online' else 'Client Batch Size'
-        first_column_tag = 'concurrency-range' if self._mode == 'online' else 'batch-size'
+        if self._was_measured_with_request_rate(measurements[0]):
+            first_column_header = 'Request Rate' if self._mode == 'online' else 'Client Batch Size'
```

Comment:
Do we have any protection for the 'else' statements (offline case)?

Reply:
We have no unit testing in place for detailed reporting. Just the  `L0_results` test that checks for PDF generation. We have an old story to add unit testing, but it really hasn't been high priority.